client/src/config.js

```diff
@@ -0,0 +1,6 @@
+const serverAdress = "localhost:1337"
+
+export default {
+    serverUrl: `http://${serverAdress}`,
+    websocketUrl: `ws://${serverAdress}`
```

Comment:
why websocket and not socket.IO ?

Reply:
Honstely I asked myself this question before I started, and consider the options, when i found this post 
https://stackoverflow.com/questions/10112178/differences-between-socket-io-and-websockets
and looked at the third comment.
I saw how simple it is and also the drawbacks where no so bad, doesn't not support IE? half the things we use also don't, it's easier to write socket.io? it's prerry easy to use websocket. and it is also fater the socket.io.
decided to use the base instead of using a wrapper, but again TBH i could as easly gone with socket.io.